RootTools/python/samples/samples_13TeV_RunIIFall17MiniAOD.py

```diff
@@ -2,6 +2,28 @@
 from CMGTools.RootTools.samples.ComponentCreator import ComponentCreator
 kreator = ComponentCreator()
 
+XS_to_be_definied = 1
+
+# Zinv
+# Cross section values from samples_13TeV_RunIISummer16MiniAODv2 -> Same process, same energy and same MC generator
```

Comment:
but different tune, thus different generator parameters (in particular alphaS and pdfs), which also affect the cross sections.
once you've added `ZJetsToNuNuHT` to the `mcSamples` please do `python samples_13TeV_RunIIFall17MiniAOD.py genXSecAna ZJetsToNuNu`

Reply:
Sample ZJetsToNuNu_HT100to200: XS(sample file) = 344.978 pb, ...  XS(genAnalyzer) = 303.3 +/- 1.059 pb :  kFactor = 1.13742 OK?  f(negw): 0.001 +- 0.000 
Sample ZJetsToNuNu_HT200to400: XS(sample file) = 96.3828 pb, ...  XS(genAnalyzer) = 93.54 +/- 0.5501 pb :  kFactor = 1.03039 OK?  f(negw): 0.001 +- 0.000 
Sample ZJetsToNuNu_HT400to600: XS(sample file) = 13.4562 pb, ...  XS(genAnalyzer) = 13.11 +/- 0.06931 pb :  kFactor = 1.02641 OK?  f(negw): 0.002 +- 0.000 
Sample ZJetsToNuNu_HT600to800: XS(sample file) = 3.96183 pb, ...  XS(genAnalyzer) = 3.251 +/- 0.01312 pb :  kFactor = 1.21865 OK?  f(negw): 0.003 +- 0.000 
Sample ZJetsToNuNu_HT800t1200: XS(sample file) = 1.81302 pb, ...  XS(genAnalyzer) = 1.499 +/- 0.007063 pb :  kFactor = 1.20949 OK?  f(negw): 0.004 +- 0.000 
Sample ZJetsToNuNu_HT1200to2500: XS(sample file) = 0.441078 pb, ...  XS(genAnalyzer) = 0.3428 +/- 0.001805 pb :  kFactor = 1.28669 OK?  f(negw): 0.007 +- 0.000 
Sample ZJetsToNuNu_HT2500toInf: XS(sample file) = 0.0100897 pb, ...  XS(genAnalyzer) = 0.004881 +/- 0.0003781 pb :  kFactor = 2.06714 ERROR  f(negw): 0.018 +- 0.000 

Can I use kFactor = 2.06714 (ERROR)?

Reply:
Also, should I use XS from sample or from genAnalyzer?